src/domain/prices.ts

```diff
@@ -158,7 +181,7 @@ function getCandlesFromPrices(prices, period) {
   }));
 }
 
-function getChainlinkChartPricesFromGraph(tokenSymbol, period) {
+export function getChainlinkChartPricesFromGraph(tokenSymbol, period) {
   if (["WBTC", "WETH", "WAVAX"].includes(tokenSymbol)) {
     tokenSymbol = tokenSymbol.substr(1);
   }
```

Comment:
oh, i think there was bug – it doesn't handle "BTC.b", could you please fix it here?

Reply:
Sure!

Reply:
Done